.github/workflows/_matrix-and-codecov-on-merge-to-main.yml

```diff
@@ -56,8 +56,7 @@ jobs:
 
       - name: Install ${{ inputs.project }} and requirements
         run: |
-          conda install --file requirements/run.txt
-          conda install --file requirements/run-ext.txt
```

Comment:
https://github.com/Billingegroup/release-scripts/pull/56#issuecomment-2386702356

as you've mentioned, I think adding `run-ext` adds an unwanted complexity. 

What if we both include optional and required dependencies all under `conda.txt` instead? 

We should test our software with all the dependencies installed anyway.. right?

Reply:
Furthermore, if we decide to use have a separate .txt file for ext pacakges, then we also needed `pip-ext.txt` I don't think we want this..